llvm/test/CodeGen/RISCV/rvv/rvv-cfi-info.ll

```diff
@@ -3,122 +3,132 @@
 ; RUN:   | FileCheck -check-prefix=OMIT-FP %s
 ; RUN: llc -mtriple=riscv64 -mattr=+v,+m -verify-machineinstrs -frame-pointer=all < %s \
 ; RUN:   | FileCheck -check-prefix=NO-OMIT-FP %s
+; RUN: llc -mtriple=riscv64 -mattr=+v,+m,+zcmp -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=OMIT-FP-ZCMP %s
+; RUN: llc -mtriple=riscv64 -mattr=+v,+m,+zcmp -verify-machineinstrs -frame-pointer=all < %s \
+; RUN:   | FileCheck -check-prefix=NO-OMIT-FP-ZCMP %s
 
 define riscv_vector_cc <vscale x 1 x i32> @test_vector_callee_cfi(<vscale x 1 x i32> %va) {
 ; OMIT-FP-LABEL: test_vector_callee_cfi:
 ; OMIT-FP:       # %bb.0: # %entry
-; OMIT-FP-NEXT:    addi sp, sp, -16
-; OMIT-FP-NEXT:    .cfi_def_cfa_offset 16
+; OMIT-FP-NEXT:    addi sp, sp, -48
```

Comment:
This code change looks highly suspicious for the change since a) it's not CFI, and b) it's a non-zcmp configuration.  My guess is that you made a mistake when rebasing?

Reply:
I added a GPR CSR clobber to an existing test.